### ISSUE TITLE
Slim CLI?

### ISSUE THREAD

**Issue by ABGEO** [slimphp/Slim-Console#1]
Hello,

I think it would be nice to create a new Slim component - **CLI Console** (like [Symfony Console](https://github.com/symfony/console)), to start the PHP Dev server, Debug router and Container, Generating Routes and Actions etc.

If you agree with me, I can immediately start working on it.

**Comment by l0gicgate** [comment]
@zhorton34 

> How cool would it be if Slim never lost it's unbiased view point, but made it extremely easy to setup preset and make commands so once a development team inevitably decides on an implementation that even if the slim preset/make console package that it was easy enough to make that they would create it themselves.

I agree, I think we should make something that is framework agnostic but that can easily be integrated with some popular libraries easily.

For now let's try and focus on one command at a time and have some thorough discussions around them. I will open separate issues for each. Let's continue the conversations in those threads

**Comment by l0gicgate** [comment]
See the two first issues:
https://github.com/slimphp/Slim-Console/issues/1
https://github.com/slimphp/Slim-Console/issues/2

Let's start there
